format-specs/geoparquet.md

```diff
@@ -50,57 +50,100 @@ Version of the geoparquet spec used, currently 0.3.0
-| encoding | string | **REQUIRED** Name of the geometry encoding format. Currently only 'WKB' is supported. |
-| geometry_type | string or \[string] | **REQUIRED** The geometry type(s) of all geometries, or 'Unknown' if they are not known.  |
-| crs       | string   | **OPTIONAL** [WKT2](https://docs.opengeospatial.org/is/18-010r7/18-010r7.html) string representing the Coordinate Reference System (CRS) of the geometry. If the crs field is not included then the data in this column must be stored in longitude, latitude. In the case where a crs is not provided, CRS-aware implementations should assume a default value of [OGC:CRS84](https://www.opengis.net/def/crs/OGC/1.3/CRS84) (longitude-latitude coordinates). |
-| orientation | string | **OPTIONAL** Winding order of exterior ring of polygons. If present must be 'counterclockwise'; interior rings are wound in opposite order. If absent, no assertions are made regarding the winding order.
-| edges | string | **OPTIONAL** Name of the coordinate system for the edges. Must be one of 'planar' or 'spherical'. The default value is 'planar'.  |
-| bbox   | \[number] | **OPTIONAL** Bounding Box of the geometries in the file, formatted according to [RFC 7946, section 5](https://tools.ietf.org/html/rfc7946#section-5). |
-| epoch    | double | **OPTIONAL** Coordinate epoch in case of a dynamic CRS, expressed as a decimal year.  |
-
+| Field Name    | Type                | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| ------------- | ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| encoding      | string              | **REQUIRED** Name of the geometry encoding format. Currently only 'WKB' is supported.                                                                                                                                                                                                                                                                                                                                                                        |
```

Comment:
Small styling nit, but I would keep the `|` as they were (without all the extra whitespace). Markdown (at the github flavored one) doesn't require this to be rendered correctly, and give this is so wide, IMO it also doesn't provide any benefit in an editor, but rather makes it harder to read (since it now gets wrapped over multiple lines)